lib/global-styles.php

```diff
@@ -88,8 +88,10 @@ function_exists( 'gutenberg_is_edit_site_page' ) &&
 		$context = 'site-editor';
 	}
 
+	// So far, only mobile will use the endpoint.
+	// However, we should be more specific here.
 	if ( defined( 'REST_REQUEST' ) && REST_REQUEST ) {
-		$context = 'rest-request';
+		$context = 'mobile';
```

Comment:
If we want to be more specific we could so something like:

```php
if (
    defined( 'REST_REQUEST' ) &&
    REST_REQUEST &&
    isset( $_GET['context'] ) &&
    'mobile' === $_GET['context']
) {
    $context = 'mobile';
}
```

If we make the API request with `context=mobile`. What do you think?

Reply:
Much better!  c521a07